tests/rbd/rbd_utils.py

```diff
@@ -511,6 +520,22 @@ def image_unmap(self, device_name):
         """UnMap provided rbd image."""
         return self.exec_cmd(cmd=f"rbd unmap {device_name}", output=True)
 
+    def benchwrite(self, **kw):
```

Comment:
Please add a small description for the method benchwrite

Reply:
Done.